src/test/run-pass/atomic-print.rs

```diff
@@ -30,9 +31,13 @@ fn main(){
         do_print(2);
         t.join();
     } else {
+        println!("{:?}", env::args());
```

Comment:
maybe remove these stray printlns?

Reply:
oh, I see that is done in a later commit

Reply:
I have cleaned up the git history